benchmarks/imagenet/resnet50/finetune_eval.py

```diff
@@ -117,13 +95,12 @@ def finetune_eval(
         ],
         logger=TensorBoardLogger(save_dir=str(log_dir), name="finetune_eval"),
         precision=precision,
-        strategy="ddp_find_unused_parameters_true",
-        num_sanity_val_steps=0,
```

Comment:
Why remove `num_sanity_val_steps`?

Reply:
I believe there was a problem with the warmup schedule or validation metrics if the sanity check ran through. This is why we disabled it. Should have made a comment (maybe there is one in the PR that added it, you can check git blame).

I would keep it for now to avoid accidentally breaking anything.